components/ListItem.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
```

Comment:
Can you remove this file? I created it and deleted it, but I think it's getting added back in because you branched off fe/make-home-page